pkg/osquery/tables/crowdstrike/falconctl/table_test.go

```diff
@@ -62,26 +82,14 @@ func TestOptionRestrictions(t *testing.T) {
 
 			// We've overridden exec with a noop, but we can check that we get the expected error back
 			_, err := testTable.generate(context.TODO(), mockQC)
-
-			if tt.expectedErr {
-				require.Error(t, err)
-				require.Contains(t, err.Error(), "requested option not allowed")
```

Comment:
I think it's worth detecting this error

Reply:
the function no longer returns and error, but we do verify that the it logs the disallowed option

Reply:
I was thinking about the log, yeah.

But I missed the addition of `expectedLog`. oops.

`expectedLog` feels slightly at odds with `expectedExecs`. Do you think it makes sense to keep it as is? Remove `expectedExecs`? Add `expectedDisallows`?

Reply:
I dropped expected log and now have `expectedExecs int` & `expectedDisallows int`, take a look, I think it makes the test easier to understand